web/viewer.html

```diff
@@ -27,6 +27,8 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+    <!-- Ignore legacy IE compatibility settings and prefer HTML5 -->
```

Comment:
I suggest to remove this comment. Everyone who is familiar with IE knows what the meta tag does. And otherwise it is just one internet search away.